data_structures/binary_tree/red_black_tree.py

```diff
@@ -150,7 +161,7 @@ def remove(self, label: int) -> RedBlackTree:
                 # it and remove that.
                 value = self.left.get_max()
                 self.label = value
-                self.left.remove(value)
+                self.left.remove(value) if value else None

```

Comment:
This should be a regular if condition because we are not capturing the value. 
```
if value:
    self.left.remove(value)
```


Questions:
1.  Should we be checking for `is None` rather than a false value of `value`?
2. When will `self.left.get_max()` not return a value?

Reply:
> This should be a regular if condition because we are not capturing the value.

@spazm Yes, you're right!

Reply:
> Questions:
> 
> 1. Should we be checking for `is None` rather than a false value of `value`?
> 2. When will `self.left.get_max()` not return a value?

1. Yes, changing to: `if value is not None` because `0` is Falsy and the `value` is an ` int`.
2.  `self.left.get_max()` can return `None` because ` self.label ` is set to `None` by default.

Reply:
I like the update. :+1: